ocaml/typing/untypeast.ml

```diff
@@ -138,6 +138,7 @@ let constant = function
   | Const_int64 i -> Pconst_integer (Int64.to_string i, Some 'L')
   | Const_nativeint i -> Pconst_integer (Nativeint.to_string i, Some 'n')
   | Const_float f -> Pconst_float (f, None)
+  | Const_float32 f -> Pconst_float (f, Some 's')
```

Comment:
I think in C this is `f`, right?  I can't recall if we discussed what the best choice here is.

Reply:
We can't use `f` because it conflicts with hex float literals, so we decided to go with `s` for single and later on `h` for half.

Reply:
Right, thanks.